ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -1253,6 +1252,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-lifecycle-advanced
+- as: aws-rosa-hcp-upgrade
+  cron: 31 13 24 * *
```

Comment:
I don't think we need cron here. Can it pass the pipeline if no cron?

Reply:
Let me try it

Reply:
It is required to add it. And I confirmed with DPTP team too